packages/backend/README.md

```diff
@@ -21,3 +21,16 @@ In order to build the docker container run the following command.
 ```shell
 yarn docker build backend -t rafiki-backend
 ```
+
+## Configuration
+
+### Redis connection
+
+The connection can be configured by specifying the following environment variables.
+The config is passed to `ioredis` - see https://github.com/luin/ioredis#tls-options.
+| Variable | Description | Default |
+|-----------------------------|------------------|
+| REDIS_URL | "redis://127.0.0.1:6379" |
```

Comment:
`REDIS_URL` (and the row above) are missing a `Description` column

Reply:
thanks, good catch